app/grandchallenge/algorithms/templates/algorithms/algorithmpermissionrequest_form.html

```diff
@@ -3,6 +3,10 @@
 {% load clean from bleach %}
 {% load guardian_tags %}
 
+{% block title %}
+    {% get_obj_perms request.user for algorithm as "algorithm_perms" %}
+    {% if "change_algorithm" in algorithm_perms %}Review Access Request{% else %}Request Access{% endif %} - {{ algorithm.title }} - {{ block.super }}
```

Comment:
`get_obj_perms` is block-scoped, so you indeed need to call it twice if you want to use it in both the title and the main content block. It does seem rather inefficient to do that though, so I'm wondering whether it's really necessary, or whether a more generic title "Access Request" would suffice. I understand that the idea of this PR is to make the title identical to the breadcrumbs, so fine for now. But for round 2 of the overarching PR, I think it's worth thinking about this. You could check the perms once and then add a context variable that you check instead.

Reply:
I agree with that. Since this part of the task is to make sure the titles are consistent with breadcrumbs, it should be fine for this PR, but definitely to be address in the title fixing PR. I will copy this note to my notes to be address later. Is that okay?